### ISSUE TITLE
Restructuring and modifying Using.md

### ISSUE THREAD

**Issue by Preeticp** [minishift/minishift#450]
We need to restructure and modify Using.md document to more accurately reflect Minishift's role as well as split the document at natural break points. The doc is already quite huge and the addition of PR #449 will add material to the split doc.

**Comment by thatdocslady** [comment]
Excellent idea to track things here, I would recommend maybe waiting until we do the asciidoc conversion (or at least you can throw ideas in here and then I can get some head start while I do the conversion, to be followed by collaborating on it further). WDYT?

**Comment by Preeticp** [comment]
@thatdocslady , when do you plan to start with the Ascidoc conversion? Sure let's collaborate and work on this.
This issue emerged out of discussions on Persistent configuration section with @hferentschik 

I see three tasks emerging from the discussion and did not intend to track any more for this issue, for now.  
1. Rewording the first section to capture the exact nature of the commands and what exactly they do, renaming and so on
2. A simple split of the doc and renaming the split doc. This is already worked out and not much to be done here.
3. A proof-read of the mounting custom shared folders. This arose only after #449 was merged. This could be separated and created as an independent issue/clubbed with this.

We can think of further splits or raise issues as and when more content comes in, once the asciidoc conversion is done.
So currently, we are talking about one more doc, let's work it out so that it's least troublesome.

**Comment by hferentschik** [comment]
Why would this depend on the Asciidoc conversion. I assume that we first get the new tooling in place and then do the (automated) conversion. If this in in place by then no problem, right? Also, I would also think that the tooling can render Markdown and Asciidoc. After all the generated docs will stay in Markdown. So I would suggest we get tooling on place without changing anything in the docs yet. We can just render HTML from Markdown. Next we start the conversion to Asciidoc for the documents we want to convert. I'd really like to separate the tooling from actual content conversion.

**Comment by thatdocslady** [comment]
Following discussion with @Preeticp , points 1+2 will be done before the asciidoc conversion, and point 3 isn't ready to document anyway so it will be deferred. 

@hferentschik to your question, a reminder that we already spoke about it when we set the tooling mission going, the manually-authored content needs to be converted to asciidoc anyway and stored in gh because of conditional text and content reuse with downstream docs. The automated conversion to asciidoc within the container will be done only to the command reference. I will wait until the tooling is in place to do so but it'll still need to be done afterwards.

**Comment by Preeticp** [comment]
@thatdocslady, Point 3 is already in the Using.md doc, and can be done as a separate issue. I pointed this comment as reference for this.

**Comment by hferentschik** [comment]
Merged via pull request #473
